sdk-tests/src/test/java/io/dapr/it/tracing/http/OpenTelemetryInterceptor.java

```diff
@@ -16,15 +16,14 @@
 import io.opentelemetry.api.OpenTelemetry;
 import io.opentelemetry.context.Context;
 import io.opentelemetry.context.propagation.TextMapPropagator;
-import javax.annotation.Nullable;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 import org.springframework.web.servlet.HandlerInterceptor;
 import org.springframework.web.servlet.ModelAndView;
 
-import javax.servlet.DispatcherType;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
+import jakarta.servlet.DispatcherType;

```

Comment:
Why not javax? Is it required because of Java17?

Reply:
Yes, Java 17 removed `javax.*` and required to use `jakarta.*`